src/test/sigrt.c

```diff
@@ -15,7 +15,7 @@ static void handle_sigrt(int sig) {
   ++num_signals_caught;
 }
 
-static void my_raise(int sig) {
+static void __attribute__ ((noinline)) my_raise(int sig) {
```

Comment:
Not that I can tell, though I didn't really try (only whatever is done by default).